From 4d90ebe52d78a11dab24f8aabb63d6936dd91fb2 Mon Sep 17 00:00:00 2001
From: Hector Banos <extern.hector.banos-marin@tttech.com>
Date: Mon, 11 Dec 2017 16:50:05 +0100
Subject: [PATCH] ADAS-145: set MAC_HFR0_SMASEL_Mask to 0.

Signed-off-by: Hector Banos <extern.hector.banos-marin@tttech.com>
---
 DWC_ETH_QOS_yregacc.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/DWC_ETH_QOS_yregacc.h b/DWC_ETH_QOS_yregacc.h
index 086501a..0a5b321 100644
--- a/DWC_ETH_QOS_yregacc.h
+++ b/DWC_ETH_QOS_yregacc.h
@@ -18928,7 +18928,7 @@
 
 /*#define MAC_HFR0_SMASEL_Mask (ULONG)(~(~0<<(1)))*/
 
-#define MAC_HFR0_SMASEL_Mask (ULONG)(0x1)
+#define MAC_HFR0_SMASEL_Mask (ULONG)(0x0)
 
 #define MAC_HFR0_SMASEL_UdfRd(data) do {\
 		MAC_HFR0_RgRd(data);\
-- 
2.7.4

